soak-tests-common/src/main/java/com/hazelcast/jet/tests/common/AbstractSoakTest.java

```diff
@@ -64,6 +64,8 @@ protected final void run(String[] args) throws Exception {
             config.addCacheConfig(cacheConfig);
             config.setJetConfig(new JetConfig().setEnabled(true));
 
+            config.setLicenseKey(System.getenv("HZ_LICENSEKEY"));
```

Comment:
This is not needed since we're using Hazelcast OS.

Reply:
Yes, I was thinking if to remove it or not. On one hand it can be useful if someone runs a test from within a local environment, and has its own copy of Enterprise. However according to this assumption, one would need to set license. 
Also another way around would be to run enterprise independently with a client connecting to it. (setting environment)

 I will remove it  - since we considered these tests as a client-specific only. (normally server is configured from files)